specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1221 @@
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateEndpointConnections/{privateEndpointName}": {
+      "get": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Get the metadata of a Private Endpoint Connection for the IoT Central Application.",
+        "operationId": "PrivateEndpoints_Get",
+        "x-ms-examples": {
+          "PrivateEndpoints_Get": {
+            "$ref": "./examples/PrivateEndpoints_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The private endpoint connection for the IoT Central Application.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Update a Private Endpoint Connection.",
+        "operationId": "PrivateEndpoints_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "PrivateEndpoints_Update": {
+            "$ref": "./examples/PrivateEndpoints_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          },
+          {
+            "name": "PrivateEndpoint",
+            "in": "body",
+            "description": "The Private Endpoint metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
needs to be marked as a long running operation

Reply:
how is that done?